unit_tests/test_fuzz_logger.py

```diff
@@ -178,15 +178,15 @@ def test_error_count(self):
         self.assertEqual(["FAILURE 1"], self.logger.error_test_cases["a"])
 
         # Use a number to verify that non-string keys can work
-        self.logger.open_test_case(test_case_id=5, name=self.some_other_text, index=self.some_int)
+        self.logger.open_test_case(test_case_id="5", name=self.some_other_text, index=self.some_int)
```

Comment:
The comment implies that a number is intentional for this test. The idea is that if one passes in a plain number, it will get converted to a string. Maybe the code in open_test_case can be changed somehow to flag to pytype that int and str are allowed?

Reply:
Or maybe the number more harmful than I realize...

Reply:
Good point, I didn't even notice the comment... must have been late that day :D

I added a type hint to FuzzLogger and reverted the changes to test_fuzz_logger.